### PR TITLE
Updating variation docs

### PR DESCRIPTION
**Pull request by jamie-m-a**
Number of updates to various docs:

Sandbox to view: http://wp-np2-11.ebi.ac.uk:7070/info/genome/variation/index.html

### CODE REVIEW

File: ensembl/htdocs/info/genome/variation/species/populations.html [hunk 1]
```diff
@@ -213,7 +213,7 @@ <h3>Populations from  the <a href="http://www.1000genomes.org" target="_blank" s
       Variants which have been discovered in this project have the "evidence status" <a href="../prediction/variant_quality.html#evidence_status"><b>1000Genomes</b></a>.
       On the website this corresponds to the icon <a href="../prediction/variant_quality.html#evidence_status"><img class="_ht" src="/i/val/evidence_1000Genomes.png" title="1000Genomes" style="vertical-align:bottom"/></a>.
       </p>
-      <h3>Population from <a href="https://grch38.togovar.org/doc/datasets/gem_j_wga" target="_blank" style="text-decoration:none">GEM-J</a></h3>
+      <h3>Population from <a href="https://togovar.biosciencedbc.jp/doc/datasets/gem_j_wga" target="_blank" style="text-decoration:none">GEM-J</a></h3>
```
[the remark this answers is not in the document]
Comment (jamie-m-a) [reply ensembl/htdocs/info/genome/variation/species/populations.html:216]: Agreed, fixed. It was generated by the script so probably an update needed.

File: docs/htdocs/info/docs/tools/vep/script/vep_plugins.html [hunk 2]
```diff
@@ -80,73 +80,76 @@ <h2 id="plugins_existing">Existing plugins</h2>
...
... to be a substitute for professional medical advice, diagnosis, or treatment, and does <br />not constitute medical or other professional advice.</p><p>Before running the plugin for the first time, you need to create a tabix index (requires tabix to be <br />installed).</p><p><pre class="code sh_sh">tabix -s 1 -b 2 -e 2 -f -S 1 AlphaMissense_hg38.tsv.gz<br /></pre><pre class="code sh_sh">tabix -s 1 -b 2 -e 2 -f -S 1 AlphaMissense_hg19.tsv.gz<br /></pre></p><p>Options are passed to the plugin as key=value pairs:<br />file             : (mandatory) Tabix-indexed AlphaMissense data<br />cols             : (optional) Colon-separated columns to print from<br />AlphaMissense data; if set to 'all', all columns are printed<br />(default: 'Missense_pathogenicity:Missense_class')<br />transcript_match : Only print data if transcript identifiers match those from<br />AlphaMissense data (default: 0)</p><p>AlphaMissense predictions are matched to input data by genomic location and protein change by default.</p></div></td><td><div class="vdoc_dtype_count" style="float:left;padding:2px 6px;cursor:default;background-color:#1E90FF">Pathogenicity predictions</div></td><td>-</td><td>Ensembl</td></tr>
```
[the remark this answers is not in the document]
Comment (jamie-m-a) [reply docs/htdocs/info/docs/tools/vep/script/vep_plugins.html:83]: Very strange don't know what happened here, I pulled from the repo after Nuno's alpha PRs were merged, in order to get the latest version of the text. I think this was also affected by whatever broke the plugin on the live site.

File: docs/htdocs/info/docs/tools/vep/script/vep_download.html [hunk 3]
```diff
@@ -116,7 +116,43 @@ <h2 id="new">What's new?</h2>
...
+        <li><a href="//github.com/Ensembl/VEP_plugins/blob/release/[[SPECIESDEFS::ENSEMBL_VERSION]]/RiboseqORFs.pm" rel="external">RiboseqORFs</a> - uses a standardized catalog of human Ribo-seq ORFs to re-calculate consequences for variants located in these translated regions</li>
+      </ul>
+
+    <li>New plugins (supported on CLI): </li>
+      <ul>
+        <li><a href="//github.com/Ensembl/VEP_plugins/blob/release/[[SPECIESDEFS::ENSEMBL_VERSION]]/Paralogues.pm" rel="external">Paralogues</a> - fetches variants overlapping the genomic coordinates of amino acids aligned between paralogue proteins</li>
+      </ul>
+
+      <li>Plugin support added to REST and Web for: </li>
+      <ul>
```
[the remark this answers is not in the document]
Comment (jamie-m-a) [reply docs/htdocs/info/docs/tools/vep/script/vep_download.html:142]: Added

File: docs/htdocs/info/docs/tools/vep/online/input.html [hunk 4]
```diff
@@ -43,7 +43,8 @@ <h2 id="input">Data input</h2>
                   
                   <ul>
                       <li><b>File upload</b> - click the "Choose file" button
-                      and locate the file on your system</li>
+                      and locate the file on your system. Please ensure large files
+                      are correctly sorted</li>
```
[the remark this answers is not in the document]
Comment (jamie-m-a) [reply docs/htdocs/info/docs/tools/vep/online/input.html]: Changed

File: docs/htdocs/info/docs/tools/vep/script/vep_plugins.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (jamie-m-a) [reply docs/htdocs/info/docs/tools/vep/script/vep_plugins.html]: Reverted to version with correct licence info

File: ensembl/htdocs/info/genome/variation/species/populations.html [hunk 5]
```diff
@@ -345,7 +345,7 @@ <h3>Populations from <a href="https://www.ncbi.nlm.nih.gov/snp/docs/gsr/alfa/" t
 
   <div style="margin-bottom:35px"></div>
 
-      <h3>Populations from  the <a href="https://esp.gs.washington.edu/drupal/" target="_blank" style="text-decoration:none">NHLBI Exome Sequencing Project</a></h3>
+      <h3>Populations from  the <a href="https://evs.gs.washington.edu/EVS/" target="_blank" style="text-decoration:none">NHLBI Exome Sequencing Project</a></h3>
```
[the remark this answers is not in the document]
Comment (jamie-m-a) [reply ensembl/htdocs/info/genome/variation/species/populations.html:348]: Yes both the official sites seem to have expired, will revert to dbGAP.

**Review by nuno-agostinho — CHANGES_REQUESTED** [review]
Hi @jamie-m-a, thanks for the PR! I put some suggestions to improve the documentation.

Please tell me if you need help or clarification regarding any of my comments.

File: docs/htdocs/info/docs/tools/vep/script/vep_cache.html [hunk 6]
```diff
@@ -174,12 +174,12 @@ <h3 id="cache_content">Data in the cache</h3>
       </tr>
       <tr>
```
[the remark this answers is not in the document]
Comment (jamie-m-a) [reply docs/htdocs/info/docs/tools/vep/script/vep_cache.html:175]: Thanks Nuno - I ran versus our server but I guess this gets added to the meta after handover, I'll add a not to the process. Will update our notes.
In fact looks like it was only added to core DB for 112, not present in 111.

File: docs/htdocs/info/docs/tools/vep/script/vep_plugins.html
[the remark this answers is not in the document]
Comment (jamie-m-a) [reply docs/htdocs/info/docs/tools/vep/script/vep_plugins.html:1]: Will check

File: ensembl/htdocs/info/genome/variation/species/sources_documentation.html [hunk 7]
```diff
@@ -1624,7 +1696,7 @@ <h2 id="neovison_vison" style="display:inline;color:#333">American mink<span cla
           <div class="vdoc_data_types">
             <div class="vdoc_type_style"><span class="_ht ht" title="Variant dynamically loaded from VCF file">Variant (from VCF)</span></div>
             <div class="vdoc_count_style"><span class="vdoc_dtype_count vdoc_thousand" title="Over 34,000 entries">34 K</span></div>
-            <div class="vdoc_example_style"><a href="/Neovison_vison/Variation/Explore?vf=FNWR01000001.1:750468:G_A:PRJEB26368" target="_blank" title="See a variation vcf example"><img src="/i/16/internal_link.png" alt="Link"/></a></div>
+            <div class="vdoc_example_style"></div>
```
[the remark this answers is not in the document]
Comment (jamie-m-a) [reply ensembl/htdocs/info/genome/variation/species/sources_documentation.html:1699]: Checking this, there were a lot of changes, not sure why they were made

File: docs/htdocs/info/docs/tools/vep/script/vep_cache.html (hunk 6 above)
[the remark this answers is not in the document]
Comment (jamie-m-a) [reply docs/htdocs/info/docs/tools/vep/script/vep_cache.html:175]: Was run from release/112 originally, not sure why it didn't grab that update, but yes have rerun after pulling again -  think it looks good now.

File: docs/htdocs/info/docs/tools/vep/script/vep_plugins.html
[the remark this answers is not in the document]
Comment (jamie-m-a) [reply docs/htdocs/info/docs/tools/vep/script/vep_plugins.html:1]: Was run from release/112 originally, not sure why it didn't grab that update, but yes have rerun after pulling again -  think it looks good now.

File: ensembl/htdocs/info/genome/variation/species/sources_documentation.html (hunk 7 above)
[the remark this answers is not in the document]
Comment (jamie-m-a) [reply ensembl/htdocs/info/genome/variation/species/sources_documentation.html:1699]: Yea looks like all the VCF-only species had the examples removed and not replaced (script probably deletes existing example but can't graba. replacement correctly for VCF-only).

File: docs/htdocs/info/docs/tools/vep/vep_faq.html [hunk 8]
```diff
@@ -60,7 +60,11 @@ <h2 id="general">General questions</h2>
 
 <p>&nbsp;</p>
 
+<p style="font-weight:bold;"> Q: Why am I not seeing the same variant that I uploaded in the output? </p>
+<p> Since the Ensembl 110 release, VEP by default will minimise the input allele for display in the output. To see the exact allele representation you provided, use the<a
+href="script/vep_options.html#opt_uploaded_allele"> --uploaded_allele</a> option. </p>
 
+<p>&nbsp;</p>
```
[the remark this answers is not in the document]
Comment (jamie-m-a) [reply docs/htdocs/info/docs/tools/vep/vep_faq.html:67]: Good point, I'll move it

File: ensembl/htdocs/info/genome/variation/species/sources_documentation.html (hunk 7 above)
[the remark this answers is not in the document]
Comment (jamie-m-a) [reply ensembl/htdocs/info/genome/variation/species/sources_documentation.html]: These have been added back in now

**Comment by jamie-m-a** [comment]
This one is ok to go in now - all comments addressed, but Nuno is on leave until after Easter so can't update his requested change.

**Pull request merged**